packages/lit-dev-content/site/blog/2022-02-07-eleventy.md

```diff
@@ -0,0 +1,181 @@
+---
+tags: blogPosts
+layout: blog-post.html
+title: "Eleventy + Lit"
+summary: "Announcing a new Lit Labs Eleventy plugin for static pre-rendering of Lit components"
+date: 2022-02-07
+author:
+  - al-marks
+---
+
+The Lit team is pleased to announce the first release of
+[`@lit-labs/eleventy-plugin-lit`](https://github.com/lit/lit/tree/main/packages/labs/eleventy-plugin-lit),
+a new plugin for [Eleventy](https://www.11ty.dev/) that pre-renders your Lit
+components as static HTML during your Eleventy build, and lets you re-hydrate
+them after your JavaScript loads.
+
+<center>
```

Comment:
Nit: Is this deprecated? [[ref](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center)]

Reply:
Ah really great point about RSS.